_dprhtml/js/index.js

```diff
@@ -53,6 +54,9 @@ function installGlobalHandlers() {
   });
 
   window.addEventListener('popstate', e => historyPopstateHandler(e));
+
+  //window.document.addEventListener('ready', mainInitialize);

```

Comment:
Can we remove this line? Looks like it was a first attempt that was replaced by the line below it.

Reply:
my bad - nuked it.